### DISCUSSION TITLE
Packages/com.unity.inputsystem/manual/ActionBindings

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#559] [Notes]
# Packages/com.unity.inputsystem/manual/ActionBindings

https://docs.unity3d.com/Packages/com.unity.inputsystem@latest/index.html?subfolder=%2Fmanual%2FActionBindings.html

**Comment by enetheru** [comment]
"Additionally, the first Binding that results in the Action changing phase will "consume" the input."
Only if the option "Project Settings -> Input System Package -> Enable Input Consumption" is ticket.
